### ISSUE TITLE
Compatibility with Angels-125 Storage Modules

### ISSUE THREAD

**Issue by Lisias** [TweakScale/Companion_WBI#1]
As mentioned [here](https://forum.kerbalspaceprogram.com/index.php?/topic/173833-18x-opt-reconfig-v191-opt-legacy-162-feb-4-2020/&do=findComment&comment=3736869)

For [OPT Reconfig](https://forum.kerbalspaceprogram.com/index.php?/topic/173833-18x-opt-reconfig-v191-opt-legacy-162-feb-4-2020/#comments).

**Comment by JadeOfMaar** [comment]
Angel-125's storage modules have the following module name, key name pairs:
* `WBIMultipurposeHab{}, WBIMultipurposeStorage{}, WBIConvertibleStorage{}` use **CapacityFactor** which is used as a multiplier for a reference volume key used elsewhere.
* `WBIOmniStorage{}` uses **storageVolume** which is an absolute unit (lol). Its units are liters/litres so it can reach hundreds or thousands.

Tweakscale currently has some effect on these (maybe all but OmniStorage. It didn't exist at the time I came to know this). If you scale a part then its storage amount is affected but when you change it to hold something else, its storage amount resets and it easily becomes OP or the opposite for its size.

For this reason I abstain from producing Tweakscale configs for any parts that use these modules, therefore, OPT Spaceplane Parts, and the abundance of tanks in any/all of Angel-125's mods.

**Comment by Lisias** [comment]
This looks like a "Fuel Switch", and should probably be on https://github.com/net-lisias-ksp/TweakScaleCompanion_FuelSwitches - but on the other hand, given the vast number of add'ons from Angel-125, perhaps I should add it to a "TweakScaleCompanion for Angel-125" instead?

**Comment by Lisias** [comment]
No. I will implement this on a dedicated Companion, given that WBI add'ons are tighly integrated. Moving this to the pertinent new Companion.
